recipes/adlfs/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.3.0" %}
+
+package:
+  name: adlfs
+  version: {{ version }}
+
+source:
+  fn: adlfs-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/a/adlfs/adlfs-{{ version }}.tar.gz
+  sha256: 71862ccad2d9e37f68b24efd9fb72d8c930667c8abf99fa73df0d3373c00d129
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - fsspec >=0.6.0
+    - requests >=2.22.0,<3.0
+    - versioneer >=0.18,<1.0
```

Comment:
I think versioneer is only required for run. It's only used by setup.py. Maybe it needs to be in the build requirements though.

Reply:
It is rare to see versioneer as a requirement, usually it is in the source